shared/video/_call-type-settings.md

```diff
@@ -0,0 +1,111 @@
+
+| Setting Name | Type   | Description         |
+| ------------ | ------ | ------------------- |
+| `width`      | Number | The width in pixels |
+| `height`     | Number | The hight in pixels |
+| `bitrate`    | Number | The bitrate         |
+
+### Screensharing
+
+| Setting Name             | Type    | Description                                                                      |
+| ------------------------ | ------- | -------------------------------------------------------------------------------- |
+| `enabled`                | Boolean | Defines whether screensharing is enabled                                         |
+| `access_request_enabled` | Boolean | When true users that do not permission to this feature can request access for it |
+
+### Recording
+
+| Setting Name | Type                                                                                                                    | Description                                                                                                                                                    |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `mode`       | String `available`, `disabled` or `auto-on`                                                                             | available → recording can be requested <br />disabled → recording is disabled <br />auto-on → recording starts and stops automatically when user join the call |
+| `quality`    | String `audio-only`, `360p`, `480p`, `720p`, `1080p`, `1440p`                                                           | Defines the resolution of the recording                                                                                                                        |
+| `audio_only` | `boolean`                                                                                                               | If `true` the recordings will only contain audio                                                                                                               |
+| `layout`     | object, for more information see the [API docs](https://getstream.io/video/docs/api/recording/calls/#recording-layouts) | Confoguration options for the recording application                                                                                                            |
+
+### Broadcasting
+
+| Setting Name | Type                  | Description                             |
+| ------------ | --------------------- | --------------------------------------- |
+| `enabled`    | Boolean               | Defines whether broadcasting is enabled |
+| `hls`        | HLS Settings (object) | Settings for HLS broadcasting           |
```

Comment:
Do we have more information on this object?

Reply:
No, not yet, but we can add a link to the API docs later, because that will probably have more information